sre-recipes/recipe.py

```diff
@@ -75,6 +75,16 @@ def _get_project_id():
         if not project_id:
             logging.error("Could not retrieve project id: " + error)
         return project_id
+    
+    @staticmethod
+    def _get_external_ip():
+        Recipe._auth_cluster()
```

Comment:
I see it's a draft so not reviewing this yet but as soon as it's ready for review - remove the draft label from your PR and let us know on team chat.